lib/src/views/Calendar/Year.tsx

```diff
@@ -27,6 +28,14 @@ export const useStyles = makeStyles(
       display: 'flex',
       justifyContent: 'center',
       padding: '8px 0',
+      '&:focus, &:hover': {
+        '& $yearButton': {
```

Comment:
Would it be possible to reduce the specificity of this selector?
Why do we need two DOM nodes per year? Could we merge it to get only one? It would improve customizability and performance.

Reply:
We need container to follow the `grid ` patter of year view. 
![image](https://user-images.githubusercontent.com/16926049/85923450-cc915000-b893-11ea-8c1e-1bc83880099b.png)

But agree it could be simplified